packages/base/lua/ErrorHandling.lua

```diff
@@ -725,11 +725,12 @@ function verifyNamedTable(data)
 	end
 end
 
---- Verify whether the user has passed only the allowed arguments for a function, showing
--- a strict warning otherwise. The warning comes from ErrorHandling:unnecessaryArgumentMsg().
+--- Verify whether the user has passed only the allowed arguments for a function, removing
+-- the unnecessary arguments and showing a strict warning otherwise.
+-- The warning comes from ErrorHandling:unnecessaryArgumentMsg().
 -- It is recommended that this function should be called as early as possible, in order
 -- to show the warning before any error that might be related to it.
--- This function returns the number of unnecessary arguments found.
```

Comment:
"This function removes each unnecessary argument from # 1." (remove empty space between # and 1).